### DISCUSSION TITLE
Subscription starter slow integration

### DISCUSSION THREAD

**Discussion by kevintatou** [vercel/community#5760] [Bug Report]
### Page to Investigate

https://vercel.com/templates/next.js/subscription-starter

### Steps to Reproduce

When i try to integrate this template, i am stuck on the Completing installation page for a long time.... So i cant finish the deployment. Any ideas?

**Comment by kevintatou** [comment]
I have manged to create it once only.... But not more than that

**Comment by olivernettinger** [comment]
Is there any solution to this? I wanted to get started, but this blocks everything...
